app/src/main/java/com/example/appempty/SecondFragment.kt

```diff
@@ -19,6 +20,10 @@ class SecondFragment : Fragment() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
+        if (resources.getBoolean(R.bool.isTablet)) {
```

Comment:
This will crash the app. Navigation can not be used on the onCreate method. Use instead the onCreateView.
Also, it is better to use `findNavController().popBackStack()` if you want to go back. That way if the user press the back button the path of screens opened does not get into the wrong screen.

Reply:
https://developer.android.com/guide/navigation/navigation-navigate